src/models/PrimaryAuth.js

```diff
@@ -37,9 +37,24 @@ function (Okta, BaseLoginModel, CookieUtil, Enums) {
       }
 
       return {
-        username: ['string', true, username],
+        username: {
+          type: 'string',
+          validate: function (value) {
+            if(!value || value.length < 1) {
+              return 'Please enter a username';
```

Comment:
i18n ? also since required is set to true is there a way to set an error message for that ? can u screenshots. I just want to make sure there is only one error message displayed.